Maddori.Apple/Maddori.Apple/Screen/Home/AddFeedback/AddFeedbackViewController.swift

```diff
@@ -180,15 +177,13 @@ class AddFeedbackViewController: BaseViewController {
         
         addFeedbackScrollView.addSubview(addFeedbackContentView)
         addFeedbackContentView.snp.makeConstraints {
-            $0.edges.equalTo(addFeedbackScrollView.snp.edges)
-            $0.width.equalTo(addFeedbackScrollView.snp.width)
-            $0.height.equalTo(1180)
+            $0.width.top.bottom.equalToSuperview()
```

Comment:
p3;
요거 top, bottom은 verticalEdges로 줄일 수 있고,
leading, trailing은 horizontalEdges로 줄일 수 있을걸요?!!

그래서 여기서는 이렇게 바꿀 수 있죠
```suggestion
            $0.width.verticalEdges.equalToSuperview()
```